subtools/dna_rna_tools.py

```diff
@@ -0,0 +1,110 @@
+def is_procedure_correct(func: str) -> bool:
+    """
+    Checks if the chosen procedure belongs to the list of available options.
+
+    Keyword arguments:
+    - func — the procedure that should be checked (str).
+
+    Output: a boolean value
+    """
+    available_functions = set(('transcribe', 'reverse', 'complement', 'reverse_complement'))
+    if not func in available_functions:
+        return True
```

Comment:
1) Ты тут вроде че то редачила, но по факту коммент Нади (очень хороший) не был исправлен
2) Вообще не надо проверять булево значение и если правда то возвращать что-то другое булево. Во первых, если else то у тебя будет return None (а мы ожидаем True или False). Во вторых, можно просто сразу возвращать результат операции. 
Вот так это должно было бы быть:
```suggestion
    return func in available_functions
```